client/utils.ts

```diff
@@ -19,8 +20,21 @@ export function debounce< T, U >( callback: ( ...args: T[] ) => U, timeout: numb
 	};
 }
 
-export function redirectToLaunchpad( siteSlug: string, launchpadFlow: string ) {
-	window.location.replace( `/setup/launchpad?flow=${ launchpadFlow }&siteSlug=${ siteSlug }` );
+export function redirectToLaunchpad(
+	siteSlug: string,
+	launchpadFlow: string,
+	emailedVerifiedParam?: string
+) {
+	let launchpadRedirectionURL = `/setup/launchpad?flow=${ launchpadFlow }&siteSlug=${ siteSlug }`;
+	if ( emailedVerifiedParam ) {
```

Comment:
I am a bit confused about what's the purpose of the `emailedVerifiedParam`, looks like it's an `URLSearchParams` instance although it's typed as a string on `line 26`. If I remove it, it still works fine, maybe I am missing something

https://user-images.githubusercontent.com/20927667/193205551-e9e34eba-4819-4f72-8d04-5e9a2ae14abf.mov

Reply:
I updated the logic, I was missing the other piece to that URLSearchParams code. 
I'm checking for the `verified` query param in the /home URL. If that param exists, then we want to add it to the launchpad redirection URL, so we can see the notification on the launchpad page.